lab-anthony/lib/parse-body.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = function(req){
+  return new Promise(function(resolve, reject) {
+    if(/(POST|PUT|DELETE)/.test(req.method)) {
+      req.body = '';
+      req.on('data', function(data) {
+        req.body += data.toString();
+      });
+
+      req.on('end', function() {
+        try {
+          req.body = JSON.parse(req.body);
+          resolve();
+        } catch(err){
+          reject(err);
+        }
+      });
+      return;
+    }
```

Comment:
People diss on else blocks, but I actually think that this is a good place for one. It's what you've set up with this return statement anyway, and I think it's clearer.